docs/remote/create-dev-container.md

```diff
@@ -0,0 +1,547 @@
+
+1. Start with **Remote-Containers: Open Repository in Container** in the Command Palette (`kbstyle(F1)`). If the repository you enter does not have a `devcontainer.json` in it, you'll be asked to select a starting point.
+2. Edit the contents of the `.devcontainer` folder as required.
+3. Try it with **Remote-Containers: Rebuild Container**.
+4. If you see an error, click on **Open in Recovery Container** in the dialog that appears.
+5. Edit the contents of the `.devcontainer` folder as required in this "recovery container."
+6. Use **Remote-Containers: Reopen in Container** and jump to step 4 if you still hit problems.
+
+### Add configuration files to public or private repositories
+
+You can easily share a customized dev container definition for your project by adding `devcontainer.json` files to source control. By including these files in your repository, anyone that opens a local copy of your repo in VS Code will be automatically prompted to reopen the folder in a container, provided they have the Remote - Containers extension installed.
+
+![Dev config file reopen notification](images/containers/dev-container-reopen-prompt.png)
+
+Beyond the advantages of having your team use a consistent environment and tool-chain, this also makes it easier for new contributors or team members to be productive quickly. First-time contributors will require less guidance and hit fewer issues related to environment setup.
+
+**Alternative: Repository configuration folders**
+
+In some cases, you may want to create a configuration for a repository that you do not control or that you would prefer didn't have a configuration included in the repository itself. To handle this situation, you can configure a location on your local filesystem to store configuration files that will be picked up automatically based on the repository.
+
+First, update the **Remote > Containers: Repository Configuration Paths** [User setting](/docs/getstarted/settings.md) with the local folder you want to use to store your repository container configuration files.
+
+In the Settings editor:
+
+![Repository container folders setting](images/containers/repo-container-folder-setting.png)
+
+Next, place your `.devcontainer/devcontainer.json` (and related files) in a sub folder that mirrors the remote location of the repository. For example, if we wanted to create a configuration for `github.com/microsoft/vscode-dev-containers`, we would create the following:
+
+```text
+📁 github.com
+    📁 microsoft
+        📁 vscode-dev-containers
+           📁 .devcontainer
+```
+
+Once in place, the configuration will be automatically picked up when using any of the remote containers commands. Once in the container, you can also select **Remote-Containers: Open Container Configuration File** from the Command Palette (`kbstyle(F1)`) to open the related `devcontainer.json` file and make further edits.
+
+## Configure a sandbox for multiple projects or folders
```

Comment:
This section could also move to advanced containers instead of here. Its somewhat like the multi-container article there in that it is a variation on a theme established in the main containers article.